_raml/apps/sales/examples/get_sales_response.json

```diff
@@ -1,4 +1,6 @@
 {
+  "createdAfter": "2016-03-20T08:21:43.779Z",
+  "createdBefore": "2016-04-19T07:21:43.779Z",
   "salesPerCurrency": [
```

Comment:
create new object *sales summary* in data-types.html with attributes:
- createdAfter (string)
- createdBefore (string)
- salesPerCurrency (array of sales)